src/html/scripts/app.js

```diff
@@ -258,10 +318,7 @@ document.getElementById("save_profile").onclick = ()=>{
         data
     }));
 }
-document.getElementById("messageContainer").scrollTop = document.getElementById("messageContainer").scrollHeight;
+
 function moveChat(){
-    let temp = document.getElementById("messageContainer");
-    if((temp.scrollHeight - temp.clientHeight) <= (temp.scrollTop + 10)){
-        temp.scrollTop = temp.scrollHeight;
-    }
-}
\ No newline at end of file
+    messageContainer.scrollTo({top: messageContainer.scrollHeight});
+}
```

Comment:
Wouldn't this change to moveChat scroll it ALWAYS? The idea is to replicate Discord's way, of not scrolling if you're scrolling up, but autoscroll if you're at the bottom.

Reply:
It does not scroll it always, only when you send something

Reply:
Egro my previous point stands, it'd autoscroll on new message, completely discarding if you're scrolling the messages above or not.

Reply:
It scrolls only on your new message not other users new messages, unless I missed something and it does do that